packages/manager/apps/pci-kubernetes/src/api/hooks/node-pools.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { act, renderHook, waitFor } from '@testing-library/react';
+import { describe, it, vi } from 'vitest';
+import { getDateFnsLocale } from '@ovh-ux/manager-core-utils';
+import * as ApiNodePoolsModule from '@/api/data/node-pools';
+import {
+  deleteNodePool,
+  getClusterNodePools,
+  TClusterNodePool,
+  updateNodePoolSize,
+} from '@/api/data/node-pools';
+import {
+  useClusterNodePools,
+  useDeleteNodePool,
+  useUpdateNodePoolSize,
+} from '@/api/hooks/node-pools';
+import { wrapper } from '@/wrapperRenders';
+
+vi.mock('@ovh-ux/manager-core-utils', async () => {
+  const mod = await vi.importActual('@ovh-ux/manager-core-utils');
+  return {
+    ...mod,
+    getDateFnsLocale: vi.fn(),
+  };
+});
```

Comment:
You dont need the mock of the module here, since you spyOn the needed functions bellow